repo/meta/schema/v3-resource-schema.json

```diff
@@ -141,15 +154,18 @@
                 }
               },
               "additionalProperties": false,
-              "type": "object"
+              "type": "object",
+              "description": "CLI information for macOS."
             }
           },
           "additionalProperties": false,
-          "type": "object"
+          "type": "object",
+          "description": "Enumerates all binaries available in this package. The supported platforms are 64 bit of Windows, Linux, and macOS."
```

Comment:
You might want to drop the second sentence, since it's just repeating what's already defined in the schema. It would likely get out of sync over time.